02_rand_dataset.ipynb

```diff
@@ -33,56 +24,400 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x11c2642b0>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Seaborn can transform a DataFrame directly into a figure\n",
+    "\n",
+    "plt.figure()\n",
+    "map = sns.heatmap(df,annot=True,fmt='d',cmap='YlGnBu',cbar=False)\n",
```

Comment:
It is not recommended to overwrite reserved word `map`. I would rename it into `map_plot` or something similar.